internal/tools/aws/constants.go

```diff
@@ -132,6 +132,10 @@ const (
 	// zone ID in AWS Route53.
 	DefaultPublicCloudDNSTagValue = "public"
 
+	// HibernatingInstallationResourceRecordIDPrefix is a prefix given to AWS
+	// route53 resource records when the installation it points to is hibernating.
+	HibernatingInstallationResourceRecordIDPrefix = "[hibernating] "
```

Comment:
@gabrieljackson I guess we need this gap here?

Reply:
I added the gap to the prefix so it would always include a space before the domain name part. You can safely check for `[hibernating]` with or without the space for blackbox.

Reply:
Make sense, thanks :)